scripts/globals/summon.lua

```diff
@@ -1,241 +1,187 @@
 
-require("scripts/globals/common");
-require("scripts/globals/status");
-require("scripts/globals/msg");
+require("scripts/globals/common")
+require("scripts/globals/status")
+require("scripts/globals/msg")
 
-function AvatarPhysicalMove(avatar,target,skill,numberofhits,accmod,dmgmod1,dmgmodsubsequent,tpeffect,mtp100,mtp200,mtp300)
-    returninfo = {};
+function getSummoningSkillOverCap(summoner)
```

Comment:
Does SMN as a subjob get this benefit or is it always 0? If it's always 0, this code is fine, otherwise subjob needs to be handled.

Reply:
nah i introduce a bug, nice catch thanks